src/domain/landingPage/landingPageMeta/LandingPageMeta.tsx

```diff
@@ -1,39 +1,57 @@
 import * as React from 'react';
 import { Helmet } from 'react-helmet';
+import { useTranslation } from 'react-i18next';
+import { useLocation } from 'react-router';
 
 import { LandingPageFieldsFragment } from '../../../generated/graphql';
 import useLocale from '../../../hooks/useLocale';
+import { ROUTES } from '../../app/routes/constants';
 import { getBannerFields } from '../../banner/bannerUtils';
 import { getLandingPageFields } from '../utils';
+
 interface Props {
   landingPage: LandingPageFieldsFragment;
 }
 
 const LandingPageMeta: React.FC<Props> = ({ landingPage }) => {
   const locale = useLocale();
+  const { pathname } = useLocation();
+  const { t } = useTranslation();
 
   const { keywords, pageTitle, metaInformation } = getLandingPageFields(
     landingPage,
     locale
   );
+
   const { someImage: topBannerSoMeImage } = getBannerFields(
     locale,
     landingPage.topBanner
   );
+
   const { someImage: bottomBannerSoMeImage } = getBannerFields(
     locale,
     landingPage.bottomBanner
   );
+
   const openGraphProperties = {
     description: metaInformation,
     image: [topBannerSoMeImage, bottomBannerSoMeImage],
     title: pageTitle,
   };
 
+  const getDescription = () => {
```

Comment:
This does not seem to work when I tried. 
Also I think that the idea of the `LandingPageMeta` is to read dynamically LandingPageQuery keywords to the meta-data, so perhaps you should use `<PageWrapper decription=...>` only for static metadata. Also I think it is much easier, you dont need any conditional logic with it.

Reply:
just tested. it works. I don't think its a correct way to put the logic of the LandingPage to the PageContainer. This fix with local files is temporary, later it will come from new CMS. So it is still dynamic and still related only to landing pages.